src/ggrc/assets/javascripts/controllers/tree_view_controller.js

```diff
@@ -1445,12 +1449,17 @@ can.Control("CMS.Controllers.TreeViewNode", {
   }
 
   , draw_node: function() {
+    if (this._draw_node_in_progress) {
```

Comment:
I think you mean `_draw_node_deferred` but that is set in `init` and already resolved in `draw_placeholder` so it's no use.
I don't like another variable either but it was the cleanest approach I could come up with.